README.md

```diff
@@ -19,6 +19,7 @@ import TreeView from 'react-expandable-treeview';
 ```javascript
     {
         id: 0,
+        hasChildren: true,
```

Comment:
Why adding an `hasChildren` attribute if you can directly check the `children` attribute?
Imho this adds too much complexity on the input object's structure